Detections/MultipleDataSources/CrossCloudUnauthorizedCredentialsAccessDetection.yaml

```diff
@@ -0,0 +1,149 @@
+id: 122fbc6a-57ab-4aa7-b9a9-51ac4970cac1
+name: Cross-Cloud Unauthorized Credential Access Detection

```

Comment:
It would be good to mention the RDS scope in the name as well - so when triaging analysts understand the RDS element

Reply:
Added

Reply:
Updated As per your suggestion